src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+
+import { ListOfPublishedProfiles } from 'firefox-profiler/components/app/ListOfPublishedProfiles';
+import { storeProfileData } from 'firefox-profiler/app-logic/published-profiles-store';
+import { blankStore } from '../fixtures/stores';
+import { mockDate } from '../fixtures/mocks/date';
+
+import 'fake-indexeddb/auto';
+import FDBFactory from 'fake-indexeddb/lib/FDBFactory';
+
+function resetIndexedDb() {
+  // This is the recommended way to reset the IDB state between test runs, but
+  // neither flow nor eslint like that we assign to indexedDB directly, for
+  // different reasons.
+  /* $FlowExpectError */ /* eslint-disable-next-line no-global-assign */
+  indexedDB = new FDBFactory();
+}
+beforeEach(resetIndexedDb);
+afterEach(resetIndexedDb);
+
+describe('ListOfPublishedProfiles', () => {
+  function setup(props) {
+    const store = blankStore();
+    const renderResult = render(
+      <Provider store={store}>
+        <ListOfPublishedProfiles {...props} />
+      </Provider>
+    );
+    return renderResult;
+  }
+  it('limits the number of rendered profiles with the limit prop', async () => {
+    // Because the rendering is dependent on the timezone, all dates in this
+    // test are using a timezone-dependent format so that the rendering is the
+    // same in all timezones.
+    mockDate('4 Jul 2020 15:00'); // Now is 4th of July, at 3pm local timezone.
+
+    // 1. Add some profiles to the local indexeddb.
+    await storeProfileData({
```

Comment:
I find it a bit odd that these profiles are added with very specific information, but then the only checks are for 

`expect(container.querySelectorAll('li')).toHaveLength(3);`

It seems like the test could either be made much terser with default values on the profile data, or we could have more coverage with a snapshot test.

Reply:
Yeah, a snapshot could be nice, and would also assert that we add a link.

Reply:
Note: we already assert the content in the test for uploaded recordings, that's why I didn't do it here initially.